blogposts/quantcast-large-scale-refactoring-with-sourcegraph.md

```diff
@@ -21,4 +21,4 @@ Founded in 2006, Quantcast's engineering team had amassed thousands of repositor
 
 Learn how, Sourcegraph's [multi-repository code search](/product/code-search-navigation) makes [large scale refactoring at Quantcast](/case-studies/quantcast) systematic, safe, and efficient.
 
-<a href="/case-studies/quantcast" class="btn btn-primary mt-4">Learn more</a>
+<a href="/case-studies/quantcast-large-scale-refactoring" class="btn btn-primary mt-4">Learn more</a>
```

Comment:
Did you confirm that this fixes it? Just want to be sure it isn't just checking for quantcast _anywhere_ in the name...

Reply:
Yes. I checked locally and confirmed the page and images are loading at the preview site - https://deploy-preview-293--sourcegraph.netlify.com/